descriptors/ModuleDescriptor-template.json

```diff
@@ -131,7 +131,7 @@
   "requires": [
     {
       "id": "instance-storage",
-      "version": "7.5"
+      "version": "8.0"
```

Comment:
If there is not code change in mod-search it supports both 7.5 and 8.0 and therefore both should be listed.

Reply:
Listing both is also needed to merge this pull request before merging the mod-inventory-storage pull request.

Reply:
- [x] Done, thanks!